Functions.Templates/Templates/KafkaOutput-CSharp/.template.config/template.json

```diff
@@ -41,8 +41,8 @@
         "Password": {
             "description": "SASL password for use with the PLAIN and SASL_SCRAM",
```

Comment:
Update description to indicate this is a connection string

Reply:
As I explained last comment, it is exactly `password` and explanation is fine. Only missing point will be , in case of EventHubs, we set Connection String. I explain it on the Readme.md part. Is it enough or should we add it in here as well?

Reply:
Please update the description to include for EventHubAPI - its connection string